CHANGELOG.md

```diff
@@ -1,38 +1,56 @@
-# Changelog
+# CHANGELOG - Terraform MongoDB Atlas Provider 
```

Comment:
Not needed.  If you look at most provider changelogs (e.g. azure, gcp, aws) they don't even have a Changelog header, they just have version headers.  Just leave it as it is w/ Changelog for now.

Reply:
makes sense, flipped back to default